packages/manager/modules/cloud-connect/src/details/overview/translations/Messages_fr_FR.json

```diff
@@ -75,11 +75,11 @@
   "cloud_connect_pop_port_unblocked": "Débloqué",
   "cloud_connect_vrack_associate": "Associer un vRack",
   "cloud_connect_vrack_remove": "Supprimer un vRack",
-  "cloud_connect_extra_type": "Type",
+  "cloud_connect_extra_type": "Extra configuration",
   "cloud_connect_extra_type_bgp": "BGP",
   "cloud_connect_extra_type_network": "Static",
-  "cloud_connect_extra_bgp_neighbor_area": "ASN client",
-  "cloud_connect_extra_ip_neighbor": "IP Neighbor",
+  "cloud_connect_extra_bgp_neighbor_area": "ASN utilisateur",
+  "cloud_connect_extra_ip_neighbor": "IP Neighbour",
```

Comment:
It's not really translated ?

Reply:
Maybe use LABEL constant to use this label instead translation key

Reply:
Sorry, I just fixed a typo on the prod version